extensions/resteasy-reactive/rest-common/spi-deployment/src/main/java/io/quarkus/resteasy/reactive/spi/GeneratedJaxRsResourceBuildItem.java

```diff
@@ -8,16 +8,38 @@
  */
 public final class GeneratedJaxRsResourceBuildItem extends MultiBuildItem {
 
-    private final String name;
+    private final String binaryName;
+    private final String internalName;
     private final byte[] data;
 
     public GeneratedJaxRsResourceBuildItem(String name, byte[] data) {
-        this.name = name;
+        this.internalName = name.replace('.', '/');
+        this.binaryName = name.replace('/', '.');
```

Comment:
This change is to make things more consistent with `GeneratedClassBuildItem` and have the same methods available.